### PR TITLE
Fix failing testsuite

### PR DESCRIPTION
**Pull request by blueForman**
Update tests according to update of phpunit version

### CODE REVIEW

File: tests/DependencyInjection/LeezyPheanstalkExtensionTest.php [hunk 1]
```diff
@@ -193,11 +189,9 @@ public function testPheanstalkProxyCustomTypeNotDefined()
         $this->container->compile();
     }
 
-    /**
-     * @expectedException \RuntimeException
-     */
     public function testPheanstalkReservedName()
     {
+        $this->expectException(\RuntimeException::class);
```
Comment (armetiz) [inline tests/DependencyInjection/LeezyPheanstalkExtensionTest.php:194]: Why did you expect an exception?

Comment (armetiz) [reply tests/DependencyInjection/LeezyPheanstalkExtensionTest.php:194]: Oh sorry, you remove the annotation ! Good point !

**Comment by armetiz** [comment]
Do you know why Scrutinizer is still in "pending"?

**Comment by armetiz** [comment]
Thank you for your pull request !

**Comment by blueForman** [comment]
> Do you know why Scrutinizer is still in "pending"?

i have no idea. I was fixing only phpunit issues

**Pull request merged**